### PR TITLE
Added npm and docker support for dependabot

### PR DESCRIPTION
**Pull request by IEvangelist**
## Summary

Added npm and docker support for dependabot. https://dependabot.com/docs/config-file/#example-configyml-files

### CODE REVIEW

**Review by gewarren — APPROVED** [review]

File: .github/dependabot.yml [hunk 1]
```diff
@@ -341,3 +341,13 @@ updates:
...
+- package_manager: "javascript"
+  directory: "/samples/snippets"
+  schedule:
+    interval: daily
+  open-pull-requests-limit: 10
+- package_manager: "docker"
+  directory: "/samples/snippets"
+  schedule:
+    interval: weekly
+  open-pull-requests-limit: 10
```
Comment (Youssef1313) [inline .github/dependabot.yml:353]: Will dependabot look for project files in nested directories?
If so, this whole file can be shrinked.

**Comment by adegeo** [comment]
Should the setting use a glob pattern that searches any `snippets` folder in the entire repo? As we move things away from the `samples/snippets` and into `docs/.../folder/snippets` paths, these won't currently be picked up.

**Comment by Youssef1313** [comment]
@adegeo dependabot doesn't support wildcards AFAIK.

**Comment by IEvangelist** [comment]
This PR is no longer needed....

**Pull request closed without merging**